src/app/clients/form/formSections/DietaryRequirementCard.tsx

```diff
@@ -4,26 +4,28 @@ import { checkboxGroupToArray, onChangeCheckbox } from "@/components/Form/formFu
 import GenericFormCard from "@/components/Form/GenericFormCard";
 import { ClientCardProps } from "../ClientForm";
 
+export const dietaryReqsLabelsAndKeys: [string, string][] = [
```

Comment:
Given this is now a top level constant can with capital case it (`DIETARY_REQS_LABELS_AND_KEYS`)?

Edit: this applies in a few places.